src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -23,6 +23,7 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
```

Comment:
This is a small thing, but Google's Java style guide recommends putting import statements in alphabetical order: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements